f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml

```diff
@@ -0,0 +1,860 @@
+      //       this.mappedPeriods = this.atatUtil.mapPackagePeriods(periods);
+      var services = this.getXaasOfferings(acqPackage.selected_service_offerings, acqPackage.sys_id);
+      gs.info("SELECTED SERVICE OFFERINGS: " + JSON.stringify(services));
+
+
+      // hoth DoW generated-document payload
+      return {
+        documentType: "DESCRIPTION_OF_WORK_DOCX",
+        templatePayload: {
+          // initial award and modifications; first table at top
+          awardHistory: this.mapAwards(this.getAwardHistory(awardIds)),
+
+          // second table at top
+          contractInformation: {
+            currentContractExists: ctb(cci.current_contract_exists),
+            contractNumber: cci.contract_number.toString(),
+            contractExpirationDate: cci.contract_order_expiration_date.toString(),
+            incumbentContractorName: cci.incumbent_contractor_name.toString(),
+            previousTaskOrderNumber: cci.task_delivery_order_number.toString()
+          },
+
+          // project overview (sections 1 and 2)
+          toTitle: acqPackage.project_overview.title.toString(),
+          scope: acqPackage.project_overview.scope.toString(),
+          scopeSurge: acqPackage.requirements_cost_estimate.surge_capabilities.toString(),
+
+          // sections 3 and 4.1-3 
+          currentEnvironment: this.getCurrentEnvironmentInfo(acqPackage.current_environment),
+          selectedClassificationLevels: this.getSelectedClassificationLevels(),
+
+          // section 4.2-4.3
+          xaasOfferings: services.xaas,
+          cloudSupportPackages: services.cloudSupport,
+
+
+          // section 5 - Contract Data Requirements List (CDRL)
+          // - no data, text/table only?
+
+          // section 6 - Performance Standards 
+          // - no data, text only ?
+
+          // section 7
+          periodOfPerformance: this.getPackagePop(acqPackage.period_of_performance, this.mappedPeriods),
+          // TODO: add task/subtask selected periods and need for entire duration??
+
+          // section 8 - Security Requirements
+          // Selected Classified Levels ??
+
+          // section 9 - GFP/GFE/GFI
+          // - no data, text only ?
+
+          // section 10
+          contractConsiderations: {
+            // 10a
+            potentialConflictOfInterest: ctb(cc.potential_conflict_of_interest),
+            conflictOfInterestExplanation: cc.conflict_of_interest_explanation.toString(),
+            // 10b
+            packagingShippingNoneApply: ctb(cc.packaging_shipping_none_apply),
+            packagingShippingOther: ctb(cc.packaging_shipping_other),
+            packagingShippingOtherExplanation:
+              cc.packaging_shipping_other_explanation.toString(),
+            contractorProvidedTransfer: ctb(cc.contractor_provided_transfer),
+            // 10c Supply Chain Risk Management 
+            // - no data, just text?
+
+            // 10d Training 
+            // - no data, just text?
+
```

Comment:
Training: https://github.com/dod-ccpo/atat-snow/pull/229

Reply:
The document appears to be just text (as if the data in #229 is included elsewhere).

Reply:
Confirmed with @luakim-ccpo that it's just hard coded text and no actual data. Check marked up template in ticket.